diff --git a/node_modules/react-scripts/config/webpack.config.js b/node_modules/react-scripts/config/webpack.config.js
--- a/node_modules/react-scripts/config/webpack.config.js
+++ b/node_modules/react-scripts/config/webpack.config.js
@@ -155,12 +155,12 @@
                   'postcss-normalize',
                 ]
               : [
-                  'tailwindcss',
+                  '@tailwindcss/postcss',
                   'postcss-flexbugs-fixes',
                   [
                     'postcss-preset-env',
                     {
                       autoprefixer: {
                         flexbox: 'no-2009',
                       },
                       stage: 3,
                     },
                   ],
                 ],
*** End Patch
